documentation/cluster-controller.adoc

```diff
@@ -1,8 +1,9 @@
 == Cluster Controller
 
-The cluster controller is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble. It's also able to deploy a
-Kafka Connect cluster which connects to an existing Kafka cluster. Such a cluster can be deployed using the Source2Image OpenShift
-feature as well in order to include more connectors plugins.
+The cluster controller is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble. As part of the Kafka cluster,
+it can also deploy the topic controller which provides a way for topics management through ConfigMaps.
+The cluster controller is also able to deploy a Kafka Connect cluster which connects to an existing Kafka cluster.
+Such a cluster can be deployed using the Source2Image OpenShift feature as well in order to include more connectors plugins.
```

Comment:
"On OpenShift such a cluster can be deployed using the Source2Image feature, providing an easy way of including more connector plugins." (Or does Kafka just use the term "connector"?)